.release.sh

```diff
@@ -2,4 +2,4 @@
 set -euxo pipefail
 
 printf "%s" "$1" > /tmp/release-notes.md
-goreleaser release --release-notes /tmp/release-notes.md --rm-dist
```

Comment:
why changing `-rm-dist` to `-clean` was needed ?

Reply:
It mentioned in the output that `--rm-dist` is deprecated and should be replaced with `--clean`. I can revert this change.